Purchases/Purchasing/Purchases.swift

```diff
@@ -251,8 +251,8 @@ public typealias DeferredPromotionalPurchaseBlock = (@escaping PurchaseCompleted
                      appUserID: String?,
                      userDefaults: UserDefaults? = nil,
                      observerMode: Bool = false,
-                     platformFlavor: String? = nil,
-                     platformFlavorVersion: String? = nil,
+                     platformFlavor: String? = Purchases.platformInfo?.flavor,
```

Comment:
Maybe change the properties in `SystemInfo` to a single `PlatformInfo` value?

Reply:
Oh, that is so much better 💯